src/fontstash.h

```diff
@@ -811,6 +858,8 @@ void fonsClearState(FONScontext* stash)
 	state->font = 0;
 	state->blur = 0;
 	state->spacing = 0;
+	state->oversampleH = 2;
```

Comment:
Hmm. I'm thinking of possible ways I could have misused it. Will write back and/or post screenshot in the next few days (hopefully).